man/rhsmcertd.8

```diff
@@ -62,10 +62,10 @@ Resets the interval for checking for and replacing expired subscriptions. This v
 file are used (unless the argument is passed again).
 
 .TP
-.B -m, --max-splay-minutes=MINUTES
-Sets the maximum time to offset the checks done by this tool. This value is in minutes. The default value is 10. It may be useful in large deployments to increase this value to reduce peak load on the entitlement server these systems are registered to. Setting this value to 0 effectively turns off this feature. This interval is in effect until the daemon restarts, and then the values in the
+.B -s, --no-splay
+Disables splay. This value is in effect until the daemon is restarted. Unless passed in again, the value of "splay" from the
```

Comment:
Like the above options, it is meant to say that this option does not affect the config. It only affects the behaviour of the current run of the daemon.

Perhaps a better description would be the below:
"""
If present it disables the splay feature entirely. When not present the splay feature is on or off based on the value of the splay option in the config.
"""

Thoughts?